src/ConnectedKubernetes/ConnectedKubernetes.Autorest/custom/Set-AzConnectedKubernetes.ps1

```diff
@@ -266,6 +274,8 @@ function Set-AzConnectedKubernetes {
         ${GatewayResourceId},
 
         [Parameter(ParameterSetName='Set', Mandatory)]
+        [Parameter(ParameterSetName='SetGatewayEnabled', Mandatory)]
```

Comment:
Weren't we moving this up to the top of the parameter list?

Reply:
Áh yeah, I bet that change was lost in different branches (oops), will do in the next commit.